Common.mk

```diff
@@ -459,9 +451,30 @@ define BUILDCTL
 			$(foreach IMPORT_CACHE,$(IMAGE_IMPORT_CACHE),--import-cache $(IMPORT_CACHE)) \
 			$(if $(IMAGE_METADATA_FILE),--metadata-file $(IMAGE_METADATA_FILE),) \
 			--opt target=$(IMAGE_TARGET) \
-			--output type=$(IMAGE_OUTPUT_TYPE),oci-mediatypes=true,\"name=$(ALL_IMAGE_TAGS)\",$(IMAGE_OUTPUT); \
-	fi
-endef 
+			--output type=$(IMAGE_OUTPUT_TYPE),oci-mediatypes=true,\"name=$(ALL_IMAGE_TAGS)\",$(IMAGE_OUTPUT)
+endef
+
+define CGO_DOCKER
+	source $(BUILD_LIB)/common.sh && build::docker::retry_pull --platform $(IMAGE_PLATFORMS) $(BUILDER_IMAGE); \
+	INTERACTIVE="$(shell if [ -t 0 ]; then echo '-it'; fi)"; \
+	docker run --rm $$INTERACTIVE -w /eks-anywhere-build-tooling/projects/$(COMPONENT) $(DOCKER_USE_ID_FOR_LINUX) \
```

Comment:
Should this be `eks-anywhere-build-tooling`?

Reply:
fixed